app/pages/sections/order-items/catalogue-solutions/select/price/contextCreator.test.js

```diff
@@ -75,7 +75,7 @@ const returnedPriceArray = [{
     value: 2,
     checked: true,
   }, {
-    html: '<div>1 - 10 consultations £700 per consultation per month</div><div>11+ consultations £400 per consultation per month</div>',
+    html: '<div>1 - 10 consultations £700.00 per consultation per month</div><div>11+ consultations £400.00 per consultation per month</div>',
```

Comment:
I need to ask Lorraine if this is a problem, I believe this is the only place a price is displayed as a whole number.